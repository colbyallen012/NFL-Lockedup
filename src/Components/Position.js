import React from 'react';
import './Position.css';

const Player = ({position, arrests}) => {
  return (
    <div className='crime'>
      <h3>{position}</h3>
      <p>Arrest Count: {arrests}</p>
    </div>
  )
}

export default Player;