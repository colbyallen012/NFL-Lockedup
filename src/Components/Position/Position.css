.position {
  background-color: lightcyan;
  color: black;
  margin: 5px auto;
  padding: 10px;
  height: 25vh;
  width:  300px;
}

