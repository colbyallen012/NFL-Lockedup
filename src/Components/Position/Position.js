import React from 'react';
import './Position.css';

const Position = ({position, arrests}) => {
  return (
    <div className='position'>
      <h3>{position}</h3>
      <p>Arrest Count: {arrests}</p>
    </div>
  )
}

export default Position;