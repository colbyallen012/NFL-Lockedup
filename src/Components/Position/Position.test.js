import React from 'react';
import { shallow } from 'enzyme';
import Position from './Position';

describe('Team', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Position position='WR' arrests={200} />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})