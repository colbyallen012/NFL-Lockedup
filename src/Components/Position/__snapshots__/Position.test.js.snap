// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Team should match the snapshot 1`] = `
<div
  className="position"
>
  <h3
    className="crime-position"
  >
    WR
  </h3>
  <p
    className="crime-arrests"
  >
    Arrest Count: 
    200
  </p>
</div>
`;
